sdcm/cluster.py

```diff
@@ -3125,15 +3125,18 @@ def __init__(self, cluster_uuid=None, cluster_prefix='cluster', node_prefix='nod
 
         # NOTE: following is needed in case of K8S where we init multiple DB clusters first
         #       and only then we add nodes to it calling code in parallel.
+        azs = len(self.params.get('availability_zone').split(","))
         if add_nodes:
             if isinstance(n_nodes, list):
                 for dc_idx, num in enumerate(n_nodes):
-                    self.add_nodes(num, dc_idx=dc_idx, enable_auto_bootstrap=self.auto_bootstrap)
+                    for az_index in range(num):
+                        self.add_nodes(1, dc_idx=dc_idx, rack=az_index % azs, enable_auto_bootstrap=self.auto_bootstrap)
             elif isinstance(n_nodes, int):  # legacy type
-                self.add_nodes(n_nodes, enable_auto_bootstrap=self.auto_bootstrap)
+                for az_index in range(n_nodes):
+                    self.add_nodes(1, rack=az_index % azs, enable_auto_bootstrap=self.auto_bootstrap)
             else:
                 raise ValueError('Unsupported type: {}'.format(type(n_nodes)))
-            self.run_node_benchmarks()
+            # self.run_node_benchmarks()
```

Comment:
Put this into it's own commit

Reply:
this is mistake, shouldn't have been commited